crates/wasm-smith/src/core.rs

```diff
@@ -568,7 +570,12 @@ impl Module {
             if !keep_going {
                 break;
             }
-            self.arbitrary_rec_group(u, AllowEmptyRecGroup::Yes)?;
+            let allow_empty = if self.empty_rec_groups > 10 {
+                AllowEmptyRecGroup::Yes
+            } else {
+                AllowEmptyRecGroup::No
+            };
```

Comment:
Can we either reuse the types limit here or add a rec groups limit to the config? Hardcoding is not ideal and 10 is also super small.